Project3/index.html

```diff
@@ -0,0 +1,201 @@
+        rel="stylesheet">
+    <title>Timeline</title>
+</head>
+
+<body>
+    <h1>A day in Orlando</h1>
+    <figure id="hour1">
+        <h2>1. 6:30am</h2>
+        <a href="#hour1">
+            <img src="images/orlando1.jpg" alt="photo of two bedroom" width="300 " height="300">
+        </a>
+        <figcaption>
+            After arriving in Orlando International Airport, I ate lunch with my travel mates and then went to premium
+            outlet to shop. While we were shopping, we got what we wanted to buy before coming here and used ‘Uber’ app
+            to go to hotel we already booked. We unpacked our room and went to a swimming pool in the hotel. We were not
+            really hungry, so we chose to eat a kid meal that consisted small fries and chicken tenders. I slept earlier
+            than usual, because we needed to get up earlier to watch ‘Harry Potter World’ in the Universal Studio’s
+            famous place.
+            I had to wake up earlier to prepare myself before going to the place, so I washed my face and brushed my
+            teeth. Also, I did my makeup and I dressed up in different costumes. In addition to myself, there were
+            others who spent time styling themselves. We checked to see if we had our passports, hotel key cards and
+            ticket confirmation before we left. We used our hotel shuttle to get to Universals Studios and it took
+            15minutes to reach there.
+
+            <a href="#" class="button-reset">&times</a>
+        </figcaption>
+    </figure>
+    <figure id="hour2">
+        <h2>2. 7:30am</h2>
+        <a href="#hour2">
+            <img src="images/orlando2.jpg" alt="photo of nature with palm trees in orlando" width="300" height="300">
+        </a>
+        <figcaption>
+            The police checked our IDs and belongings after we arrived at Universals Studios parking garage. It took
+            some time until we reached the entrance of the park. I could see palm trees and all types of nature that I
+            hadn’t seen in Baltimore; and then finally we were there.
+            <a href="#" class="button-reset">&times</a>
+        </figcaption>
+    </figure>
+    <figure id="hour3">
+        <h2>3. 8:30am</h2>
+        <a href="#hour3">
+            <img src="images/orlando3.jpg" alt="photo of orlando universtudo's entry" width="300" height="300">
+        </a>
+        <figcaption>
+            I got in line at the entry for a long time and I visited the famous place called ‘the wizarding world of
+            Harry Potter’. In Harry Potter land, there were two parts of the land and I went one of the parts called
+            ‘Harry Potter and the escape Gringotts’.
+            <a href="#" class="button-reset">&times</a>
+        </figcaption>
+    </figure>
+    <figure id="hour4">
+        <h2>4. 9:30am</h2>
+        <a href="#hour4">
+            <img src="images/orlando4.jpg" alt="photo of moving harrypotter character" width="300" height="300">
```

Comment:
Since `alt` text is read aloud by a screenreader, you need to be careful about typos and maintaining proper spaces between words:
```suggestion
            <img src="images/orlando4.jpg" alt="photo of moving harry potter character" width="300" height="300">
```